source/base/Text.ooc

```diff
@@ -164,6 +169,16 @@ Text: cover {
 		separator free(Owner Receiver)
 		result
 	}
+	trim: func -> This {
+		length := this count
```

Comment:
@simonmika what it does is that it removes a function call from the loop condition, how is that bad?